assets/dev/scss/frontend/widgets/icon-list.scss

```diff
@@ -86,7 +86,7 @@
 		.elementor-icon-list-icon {
 			text-align: var(--e-icon-list-icon-align);
 
-			svg {
```

Comment:
I don't understand what is this `i` element addition

Reply:
* Fixed *